src/pip/_internal/req/req_install.py

```diff
@@ -757,10 +755,9 @@ def install(
             prefix=prefix,
         )
 
-        global_options = global_options if global_options is not None else []
         if self.editable and not self.is_wheel:
             install_editable_legacy(
-                global_options=global_options,
+                global_options=global_options if global_options is not None else [],
```

Comment:
Why is this change needed?

Reply:
Cosmetics only. Since the variable is not used anywhere else, I figured I'd inline it.